hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/interledger/ClprSuite.java

```diff
@@ -95,45 +102,49 @@ final Stream<DynamicTest> exchangesConfigurationsViaRealClient() {
                         final var storedFirst = awaitConfiguration(client, ledgerId, firstConfig);
-                        Assertions.assertEquals(firstConfig, storedFirst);
+                        assertConfigurationAtLeast(firstConfig, storedFirst);
 
                         final var secondConfig = buildConfiguration(ledgerId, secondInstant, endpoints);
                         Assertions.assertEquals(
-                                ResponseCodeEnum.SUCCESS,
+                                ResponseCodeEnum.OK,
                                 client.setConfiguration(payerAccountId, node.getAccountId(), secondConfig));
                         final var storedSecond = awaitConfiguration(client, ledgerId, secondConfig);
-                        Assertions.assertEquals(secondConfig, storedSecond);
+                        assertConfigurationAtLeast(secondConfig, storedSecond);
 
                         final var thirdConfig = buildConfiguration(ledgerId, thirdInstant, endpoints);
                         Assertions.assertEquals(
-                                ResponseCodeEnum.SUCCESS,
+                                ResponseCodeEnum.OK,
                                 client.setConfiguration(payerAccountId, node.getAccountId(), thirdConfig));
                         final var storedThird = awaitConfiguration(client, ledgerId, thirdConfig);
-                        Assertions.assertEquals(thirdConfig, storedThird);
+                        assertConfigurationAtLeast(thirdConfig, storedThird);
                     } catch (final InterruptedException e) {
                         Thread.currentThread().interrupt();
                         throw new IllegalStateException("Interrupted while awaiting CLPR configuration", e);
                     }
                 }),
-                sourcing(() -> QueryVerbs.getLedgerConfig(ledgerIdString).hasTimestamp(thirdInstant.getEpochSecond())),
-                withOpContext((spec, log) -> awaitLogContains(spec, ENDPOINT_SUCCESS_PREFIX, LOG_WAIT_TIMEOUT)),
-                withOpContext((spec, log) -> awaitLogContains(spec, BOOTSTRAP_PREFIX, LOG_WAIT_TIMEOUT)),
-                withOpContext((spec, log) -> assertLogDoesNotContain(spec, FETCH_FAILURE_MESSAGE)),
-                withOpContext((spec, log) -> assertLogDoesNotContain(spec, INVALID_STATUS_TOKEN)),
-                withOpContext((spec, log) -> assertLogDoesNotContain(spec, INVALID_BODY_STATUS_TOKEN)));
+                sourcing(() -> QueryVerbs.getLedgerConfig(ledgerIdString).hasTimestamp(thirdInstant.getEpochSecond())));
     }
 
     private static ClprLedgerConfiguration awaitConfiguration(
             final ClprClientImpl client, final ClprLedgerId ledgerId, final ClprLedgerConfiguration expected)
             throws InterruptedException {
         final long deadlineNanos = System.nanoTime() + CONFIG_APPLY_TIMEOUT.toNanos();
         ClprLedgerConfiguration current;
+        System.out.printf(
```

Comment:
Any reason not to use the standard logging mechanism for these print statements?